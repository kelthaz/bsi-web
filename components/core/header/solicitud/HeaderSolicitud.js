import React from 'react';

const HeaderSolicitud = () => {
  return <div></div>;
};

export default HeaderSolicitud;
