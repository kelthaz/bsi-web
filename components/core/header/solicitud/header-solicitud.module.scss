@import '../../../../styles/scss/modules/var';
@import '../../../../styles/scss/modules/mixins';

.container-header {
  align-items: center;
  border-bottom: 1px solid $blue-shadow;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding: 16px 32px;

  @include media('sm') {
    padding: 14px 16px;
  }
}
