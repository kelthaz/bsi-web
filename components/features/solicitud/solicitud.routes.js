import cargaDocumentosRoutes from './carga-documentos/carga-documentos.routes';
import datosEmpresaRoutes from './datos-empresa/datos-empresa.routes';
import datosPersonalesRoutes from './datos-personales/datos-personales.routes';
import documentacionRoutes from './documentacion/documentacion.routes';
import ofertaRoutes from './oferta/oferta.routes';

const solicitudRoutes = [
  ...datosPersonalesRoutes,
  // ...datosEmpresaRoutes,
  // ...documentacionRoutes,
  // ...cargaDocumentosRoutes,
  // ...ofertaRoutes
].map((route) => ({ ...route, feature: 'solicitud' }));

export default solicitudRoutes;
