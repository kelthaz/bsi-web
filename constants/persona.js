export const MORAL = 'MORAL';
export const FISICA = 'FISICA';
