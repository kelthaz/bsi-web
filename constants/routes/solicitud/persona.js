export const bienvenida = '/solicitud/datos-personales/bienvenida';
export const granSalto = '/solicitud/datos-personales/gran-salto';
export const stepOne = '/solicitud/datos-personales/1';
export const stepTwo = '/solicitud/datos-personales/2';
export const stepThree = '/solicitud/datos-personales/3';
export const stepFour = '/solicitud/datos-personales/4';
export const stepFive = '/solicitud/datos-personales/5';
export const agradecimiento = '/solicitud/datos-personales/agradecimiento';
