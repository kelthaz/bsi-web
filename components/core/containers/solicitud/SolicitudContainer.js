import { useRouter } from 'next/router';
import PropTypes from 'prop-types';
import { useEffect } from 'react';
import { useSelector } from 'react-redux';
import { DATO_PERSONA } from '../../../../constants/formularios';
import { SIMULADOR_ROUTE } from '../../../../constants/routes/publico/publico';
import solicitudRoutes from '../../../features/solicitud/solicitud.routes';
import Step from '../../../shared/step/Step';
import TabInformativo from '../../../shared/tab-informativo/TabInformativo';
import SvgPatronesSolicitud from '../../../svgs/SvgPatronesSolicitud';
import HeaderSolicitud from '../../header/solicitud/HeaderSolicitud';
import ModalActualizar from '../../modals/solicitud/modal-actualizar/ModalActualizar';

const SolicitudContainer = ({ pageComponent, servicesData }) => {
  const { component, data } = pageComponent;
  const [componentPFAE, componentPM] = component;
  const { formulario, paso, step, tab, tipoPersona } = data;
  const { sincronizado, currentStep, tipoPersonaRedux } = useSelector((state) => state.solicitud);
  const { replace } = useRouter();

  let Component = <></>;

  if (tipoPersona === '' && componentPM) {
    Component = tipoPersonaRedux === 'MORAL' ? componentPM : componentPFAE;
  } else {
    Component = componentPFAE;
  }

  const componentsFormulario = solicitudRoutes
    .filter(({ data: dataComponent }) => dataComponent.formulario === formulario)
    .map(({ data: dataComponent, route }) => ({
      step: dataComponent.step,
      paso: dataComponent.paso,
      tipoPersona: dataComponent.tipoPersona,
      action: route,
    }));

  const steps = componentsFormulario.filter(
    (dataComponent) =>
      Number.isInteger(dataComponent.step) &&
      (dataComponent.tipoPersona === '' || dataComponent.tipoPersona === tipoPersonaRedux)
  );

  const tabs = [
    { path: 'datos-personales', label: 'Datos personales' },
    { path: 'datos-empresa', label: 'Datos de tu empresa' },
    { path: 'oferta', label: 'Oferta' },
    { path: 'documentacion', label: 'DocumentaciÃ³n' },
  ];

  useEffect(() => {
    if (!sincronizado && formulario === DATO_PERSONA) {
      replace(SIMULADOR_ROUTE);
    }
  }, []);

  return (
    <>
      <HeaderSolicitud />
      <ModalActualizar />

      {!!tab && (
        <TabInformativo
          tabs={tabs}
          currentTab={tab}
          currentStep={paso}
          valipStep={currentStep.paso}
          steps={componentsFormulario}
        />
      )}

      {!!step && <Step currentStep={step} valipStep={currentStep.valipStep} steps={steps} />}
      <Component {...servicesData} validate={currentStep.paso === paso} />
      <SvgPatronesSolicitud className="fixed-left-bottom" />
    </>
  );
};

SolicitudContainer.propTypes = {
  pageComponent: PropTypes.any.isRequired,
  servicesData: PropTypes.any.isRequired,
};

export default SolicitudContainer;
