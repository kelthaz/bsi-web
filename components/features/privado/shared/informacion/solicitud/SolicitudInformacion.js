import React from 'react';
import PropTypes from 'prop-types';
import { MORAL } from '../../../../../../constants/persona';

const SolicitudInformacion = ({ informacionSolicitud, tipoPersona }) => {
  const labels = {
    usoCredito: 'Destino del crédito',
    descripcionCredito: 'Razón social',
    domicilioFiscal: 'Domicilio fiscal',
    domicilioComercial: 'Domicilio comercial',
    nombreCompletoRecibe: 'Nombre de quien podría recibir token',
    celularRecibe: 'Teléfono de quien podría recibir token',
    telefonoEmpresa: 'Teléfono del negocio',
    curp: tipoPersona === MORAL ? 'CURP' : 'CURP del representante legal',
    estadoCivil: 'Estado civil del solicitante',
    nombreConyuge: 'Nombre de la Pareja',
    numeroEmpleados: 'Número de empleados',

    montoSolicitado: 'Crédito solicitado',
    montoAprobado: 'Crédito aprobado',

    tasaOrdinaria: 'Tasa ordinaria',
    tasaMoratoria: 'Tasa moratoria',
    comisionApertura: 'Comisión por apertura',
    plazo: 'Plazo del crédito',
    periodicidad: 'Esquema de pago',
    pago: 'Pagos',
    cat: 'CAT',
    tieneCuentaBancoppel: 'Tiene cuenta BanCoppel',
    clabe: 'Número de cuenta BanCoppel',
  };

  if (tipoPersona === MORAL) {
    delete labels.estadoCivil;
    delete labels.nombreConyuge;
  }

  const itemsPerfil = Object.keys(labels);

  return (
    <>
      {itemsPerfil.map((item) => (
        <div key={item} className="row pb-3">
          <div className="col-6">
            <div className="body2">{labels[item]}</div>
          </div>
          <div className="col-6">
            <div className="color-blue-storm body2">{informacionSolicitud[item]}</div>
          </div>
        </div>
      ))}
    </>
  );
};

SolicitudInformacion.propTypes = {
  informacionSolicitud: PropTypes.object.isRequired,
  tipoPersona: PropTypes.string.isRequired,
};

export default SolicitudInformacion;
