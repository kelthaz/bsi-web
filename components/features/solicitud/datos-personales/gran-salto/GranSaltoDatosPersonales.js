import React from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { useRouter } from 'next/router';
import PropTypes from 'prop-types';
import SvgDocumentos from '../../../../svgs/SvgDocumentos';
import SvgOferta from '../../../../svgs/SvgOferta';
import SvgPersona from '../../../../svgs/SvgPersona';
import SvgPersonaMoral from '../../../../svgs/SvgPersonaMoral';
import { nextStepDatosPersonales } from '../../../../../redux/actions/solicitud';
import { PASO_UNO_DATOS_PERSONA_ROUTE } from '../../../../../constants/routes/solicitud/persona';

const GranSaltoDatosPersonales = ({ validate }) => {
  const { currentStep } = useSelector((state) => state.solicitud);
  const dispatch = useDispatch();
  const { push } = useRouter();

  const handleComencemos = async () => {
    dispatch(
      nextStepDatosPersonales({
        currentStep: validate ? { ...currentStep, paso: currentStep.paso + 1 } : { ...currentStep },
      })
    );
    await push(PASO_UNO_DATOS_PERSONA_ROUTE);
  };

  return (
    <div className="contedor-fixed">
      <div className="contedor-solicitud">
        <div className="container">
          <div className="row">
            <h2 className="color-blue-storm">
              ¡Estás por dar el siguiente <br className="only-lg-inline" /> gran salto!
            </h2>
            <p className="body2 color-gray-dark">
              En este primer paso te pediremos nos compartas un poco de información para conocerte mejor. Nuestro
              proceso se divide en cuatro bloques y te tomará aproximadamente 30 minutos completarlo todo:
            </p>
          </div>
          <div className="row no-gutters">
            <div className="col-lg-3 col-md-3 col-sm-12 col-xs-12">
              <div className="card-simple-transparent-img">
                <SvgPersona />
                <p>Datos Personales</p>
              </div>
            </div>
            <div className="col-lg-4 col-md-4 col-sm-12 col-xs-12">
              <div className="card-simple-transparent-img disabled-card-svg">
                <SvgPersonaMoral />
                <p>Datos de tu empresa</p>
              </div>
            </div>
            <div className="col-lg-2 col-md-2 col-sm-12 col-xs-12">
              <div className="card-simple-transparent-img disabled-card-svg">
                <SvgOferta />
                <p>Oferta</p>
              </div>
            </div>
            <div className="col-lg-3 col-md-3 col-sm-12 col-xs-12">
              <div className="card-simple-transparent-img disabled-card-svg">
                <SvgDocumentos />
                <p>Documentación</p>
              </div>
            </div>
          </div>
          <div className="row flex-column-start-config">
            <p className="body2 color-gray-dark">En este primer bloque deberás contar con:</p>
            <div className="card-simple-blue-light list-onboarding">
              <ul>
                <li>RFC</li>
              </ul>
            </div>
            <button type="button" className="btn-medium flex-align-self-center my-3" onClick={handleComencemos}>
              ¡Comencemos!
            </button>
          </div>
        </div>
      </div>
    </div>
  );
};

GranSaltoDatosPersonales.propTypes = {
  validate: PropTypes.bool.isRequired,
};

export default GranSaltoDatosPersonales;
