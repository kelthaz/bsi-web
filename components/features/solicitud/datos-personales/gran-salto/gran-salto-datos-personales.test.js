import '@testing-library/jest-dom';
import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { useRouter } from 'next/router';
import storeTest from '../../../../../redux/storeTest';
import GranSaltoDatosPersonales from './GranSaltoDatosPersonales';
import { PASO_UNO_DATOS_PERSONA_ROUTE } from '../../../../../constants/routes/solicitud/persona';

describe('Pruebas en el componente GranSaltoDatosPersonales', () => {
  const store = storeTest();

  const push = jest.fn();
  useRouter.mockImplementation(() => ({ push }));

  const wrapper = mount(
    <Provider store={store}>
      <GranSaltoDatosPersonales validate />
    </Provider>
  );

  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('Debe de mostrarse correctamente el componente', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('Debe de guardar en el reducer el nuevo paso y enviarlo al siguiente paso', async () => {
    // arrange
    const {
      solicitud: { currentStep },
    } = store.getState();
    const comencemos = wrapper.find('button');
    // act
    comencemos.simulate('click');
    // assert
    expect(store.getState().solicitud.currentStep.paso).toBe(currentStep.paso + 1);
    expect(push).toHaveBeenCalledTimes(1);
    expect(push).toHaveBeenCalledWith(PASO_UNO_DATOS_PERSONA_ROUTE);
  });

  test('Debe de guardar en el reducer el nuevo paso y enviarlo al siguiente paso', async () => {
    // arrange
    const {
      solicitud: { currentStep },
    } = store.getState();

    const wrap = mount(
      <Provider store={store}>
        <GranSaltoDatosPersonales validate={false} />
      </Provider>
    );
    const comencemos = wrap.find('button');
    // act
    comencemos.simulate('click');
    // assert
    expect(store.getState().solicitud.currentStep.paso).toBe(currentStep.paso);
    expect(push).toHaveBeenCalledTimes(1);
    expect(push).toHaveBeenCalledWith(PASO_UNO_DATOS_PERSONA_ROUTE);
  });
});
