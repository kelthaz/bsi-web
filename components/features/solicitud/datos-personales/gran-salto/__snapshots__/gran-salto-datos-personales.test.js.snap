// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente GranSaltoDatosPersonales Debe de mostrarse correctamente el componente 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <GranSaltoDatosPersonales
    validate={true}
  >
    <div
      className="contedor-fixed"
    >
      <div
        className="contedor-solicitud"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <h2
              className="color-blue-storm"
            >
              ¡Estás por dar el siguiente 
              <br
                className="only-lg-inline"
              />
               gran salto!
            </h2>
            <p
              className="body2 color-gray-dark"
            >
              En este primer paso te pediremos nos compartas un poco de información para conocerte mejor. Nuestro proceso se divide en cuatro bloques y te tomará aproximadamente 30 minutos completarlo todo:
            </p>
          </div>
          <div
            className="row no-gutters"
          >
            <div
              className="col-lg-3 col-md-3 col-sm-12 col-xs-12"
            >
              <div
                className="card-simple-transparent-img"
              >
                <SvgPersona>
                  <svg
                    fill="none"
                    height={48}
                    viewBox="0 0 48 48"
                    width={48}
                  >
                    <path
                      d="M22.535 11.289a7.86 7.86 0 011.179 4.168c0 4.12-3.09 7.458-6.903 7.458-2.7 0-5.039-1.676-6.172-4.117"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.5}
                    />
                    <path
                      d="M10.639 18.798a7.913 7.913 0 01-.73-3.34c0-4.12 3.09-7.458 6.902-7.458 2.384 0 4.484 1.304 5.724 3.29"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.5}
                    />
                    <path
                      d="M11.422 39.801h11.84c8.95.615 8.416-3.77 8.416-3.77-.119-3.356-3.034-5.462-6.204-6.763"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.5}
                    />
                    <path
                      d="M25.474 29.268a21.526 21.526 0 00-16.264 0c-3.17 1.301-6.084 3.407-6.204 6.763 0 0-.534 4.388 8.416 3.77M29.29 13.658h15.663M29.29 19.701h7.167"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.5}
                    />
                    <path
                      d="M29.29 25.745h14.577"
                      stroke="#FFBE12"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.5}
                    />
                  </svg>
                </SvgPersona>
                <p>
                  Datos Personales
                </p>
              </div>
            </div>
            <div
              className="col-lg-4 col-md-4 col-sm-12 col-xs-12"
            >
              <div
                className="card-simple-transparent-img disabled-card-svg"
              >
                <SvgPersonaMoral>
                  <svg
                    fill="none"
                    height={48}
                    viewBox="0 0 48 48"
                    width={48}
                  >
                    <path
                      d="M4 26.295V42.88c0 1.683 1.185 3.05 2.646 3.05h12.327"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.8}
                    />
                    <path
                      d="M23.893 26.192V6.052C23.893 4.367 22.708 3 21.247 3H6.646C5.185 3 4 4.367 4 6.052v20.243"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.8}
                    />
                    <path
                      d="M43.357 36.347v6.533c0 1.685-1.185 3.052-2.646 3.052H19.48c-1.461-.003-2.646-1.37-2.646-3.055v-7.45"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.8}
                    />
                    <path
                      d="M16.834 35.429v-6.185c0-1.686 1.185-3.052 2.646-3.052h21.234c1.46 0 2.645 1.366 2.645 3.052v7.103M7.85 10.155h5.134M7.85 15.582h5.134M7.85 21.01h5.134M7.85 26.439h5.134"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.8}
                    />
                    <path
                      d="M32.48 36.8h-3.916c-.926 0-1.679.866-1.679 1.937v7.192h7.272v-7.192c.003-1.07-.748-1.937-1.676-1.937z"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={2.8}
                    />
                  </svg>
                </SvgPersonaMoral>
                <p>
                  Datos de tu empresa
                </p>
              </div>
            </div>
            <div
              className="col-lg-2 col-md-2 col-sm-12 col-xs-12"
            >
              <div
                className="card-simple-transparent-img disabled-card-svg"
              >
                <SvgOferta>
                  <svg
                    fill="none"
                    height={48}
                    viewBox="0 0 48 48"
                    width={48}
                  >
                    <path
                      d="M0 0h48v48H0z"
                      fill="#fff"
                    />
                    <path
                      d="M16.531 15.75h14.625M16.586 23.063h10M16.586 30.063h7"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeMiterlimit={10}
                      strokeWidth={4}
                    />
                    <path
                      d="M23.844 40.125H11.018C9.35 40.125 8 38.825 8 37.221V8.904C8 7.3 9.35 6 11.018 6h12.826"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeMiterlimit={10}
                      strokeWidth={4}
                    />
                    <path
                      d="M15.313 6h21.374c1.658 0 3 1.3 3 2.904v28.317c0 1.604-1.342 2.904-3 2.904h-12.75"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeMiterlimit={10}
                      strokeWidth={4}
                    />
                    <path
                      d="M34.786 42.302c-1.12 0-2.128-.136-3.024-.408-.896-.288-1.752-.784-2.568-1.488a1.894 1.894 0 01-.48-.6 1.491 1.491 0 01-.168-.672c0-.368.128-.688.384-.96a1.331 1.331 0 011.008-.432c.304 0 .576.096.816.288.608.496 1.208.872 1.8 1.128.608.256 1.352.384 2.232.384a4.85 4.85 0 001.632-.264c.496-.192.896-.44 1.2-.744.304-.32.456-.68.456-1.08 0-.48-.144-.888-.432-1.224-.288-.336-.728-.616-1.32-.84-.592-.24-1.344-.424-2.256-.552-.864-.128-1.624-.32-2.28-.576-.656-.272-1.208-.608-1.656-1.008a4.202 4.202 0 01-.984-1.44 5 5 0 01-.336-1.872c0-1.04.264-1.928.792-2.664.544-.736 1.272-1.296 2.184-1.68.912-.384 1.92-.576 3.024-.576 1.04 0 2 .16 2.88.48.896.304 1.624.696 2.184 1.176.464.368.696.792.696 1.272 0 .352-.136.672-.408.96-.272.288-.592.432-.96.432-.24 0-.456-.072-.648-.216-.256-.224-.6-.432-1.032-.624a7.746 7.746 0 00-1.368-.504 4.693 4.693 0 00-1.344-.216c-.688 0-1.272.088-1.752.264-.464.176-.816.416-1.056.72-.24.304-.36.656-.36 1.056 0 .48.136.88.408 1.2.288.304.696.552 1.224.744.528.176 1.16.336 1.896.48.96.176 1.8.384 2.52.624.736.24 1.344.552 1.824.936.48.368.84.84 1.08 1.416.24.56.36 1.248.36 2.064 0 1.04-.288 1.936-.864 2.688-.576.752-1.336 1.328-2.28 1.728-.928.4-1.936.6-3.024.6zm1.728 1.296c0 .416-.144.76-.432 1.032a1.359 1.359 0 01-1.032.432c-.4 0-.736-.144-1.008-.432-.272-.272-.408-.616-.408-1.032V24.302c0-.416.136-.76.408-1.032.288-.288.64-.432 1.056-.432.416 0 .752.144 1.008.432.272.272.408.616.408 1.032v19.296z"
                      fill="#FFBE12"
                    />
                  </svg>
                </SvgOferta>
                <p>
                  Oferta
                </p>
              </div>
            </div>
            <div
              className="col-lg-3 col-md-3 col-sm-12 col-xs-12"
            >
              <div
                className="card-simple-transparent-img disabled-card-svg"
              >
                <SvgDocumentos>
                  <svg
                    fill="none"
                    height={48}
                    viewBox="0 0 48 48"
                    width={48}
                  >
                    <path
                      d="M35.758 13.47h7.284a1.95 1.95 0 011.959 1.942V38.73a1.95 1.95 0 01-1.959 1.943h-30.51"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={3}
                    />
                    <path
                      d="M12.211 40.674H4.025C2.906 40.674 2 39.802 2 38.726V9.274c0-1.076.906-1.947 2.025-1.947h14.768c.747 0 1.433.394 1.785 1.027l2.244 4.03a2.035 2.035 0 001.784 1.028h11.618"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={3}
                    />
                    <path
                      d="M18.674 24l4.388-4.388L27.449 24M23.06 21.367v8.776"
                      stroke="#81C1EA"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={3}
                    />
                    <path
                      d="M16.041 30.143v3.51h14.041v-3.51"
                      stroke="#225AA7"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeMiterlimit={10}
                      strokeWidth={3}
                    />
                  </svg>
                </SvgDocumentos>
                <p>
                  Documentación
                </p>
              </div>
            </div>
          </div>
          <div
            className="row flex-column-start-config"
          >
            <p
              className="body2 color-gray-dark"
            >
              En este primer bloque deberás contar con:
            </p>
            <div
              className="card-simple-blue-light list-onboarding"
            >
              <ul>
                <li>
                  RFC
                </li>
              </ul>
            </div>
            <button
              className="btn-medium flex-align-self-center my-3"
              onClick={[Function]}
              type="button"
            >
              ¡Comencemos!
            </button>
          </div>
        </div>
      </div>
    </div>
  </GranSaltoDatosPersonales>
</Provider>
`;
